modules/secrets-store-aws-systems-manager.adoc

```diff
@@ -0,0 +1,13 @@
+// Module included in the following assemblies:
+//
+// * nodes/pods/nodes-pods-secrets-store.adoc
+
+:_content-type: PROCEDURE
+[id="secrets-store-aws-systems-manager_{context}"]
+= Mounting secrets from AWS Systems Manager Parameter Store
+
+You can use the {secrets-store-operator} to mount secrets from AWS Systems Manager Parameter Store to a CSI volume in {product-title}.
+
+// TODO add the rest of the content once we nail down the other AWS procedure
+
+// TODO: Figure out where the differences in the procedures are for the two different AWS providers
```

Comment:
Test case is OCP-66374. The only difference is objectType uses ssmparameter instead of `objectType: "secretsmanager"`:
```
  parameters:
    objects: |
      - objectName: "xxiaParameter"
        objectType: "ssmparameter"
```

Reply:
@xingxingxia So you're saying that the entire procedure is exactly the same as AWS secrets manager, except for this single change?

If that's the case, then I'll reuse the same file/procedure and just add some conditionals to change just this piece.

Reply:
Also let me know if the verification steps still look the same for this objectType

Reply:
Oh, if fully showing the differences, compare OCP-66032 for secretsmanager VS OCP-66374 for parameter, 3 differences:
AWS commands different in creating the secret/parameter:
```
OCP-66032:
aws --region "$REGION" secretsmanager  create-secret --name xxiaSecret --secret-string '{"username":"memeuser", "password":"hunter2"}' --description "xxia test purpose"

OCP-66374:
aws ssm put-parameter --name "xxiaParameter" --value "xxia parameter" --type String --region "$REGION"
```

Permissions different for the serviceaccount:
```
OCP-66032:
    statementEntries:
    - action:
      - "secretsmanager:GetSecretValue"
      - "secretsmanager:DescribeSecret"
      effect: Allow
      resource: "arn:*:secretsmanager:*:*:secret:xxiaSecret-??????"

OCP-66374:
    statementEntries:
    - action:
      - "ssm:GetParameter"
      - "ssm:GetParameters"
      effect: Allow
      resource: "arn:*:ssm:*:*:parameter/xxiaParameter*"
```

SecretProviderClass `parameters` different in `objectType`:
```
OCP-66032:
  parameters:
    objects: |
      - objectName: "xxiaSecret"
        objectType: "secretsmanager"

OCP-66374:
    objects: |
      - objectName: "xxiaParameter"
        objectType: "ssmparameter"
```

Maybe you need not document details for Systems Manager Parameter Store. Just say:
```
You can use the {secrets-store-operator} to mount secrets from AWS Systems Manager Parameter Store to a CSI volume in {product-title}. Note, in SecretProviderClass, the `objectType` for AWS Systems Manager Parameter Store objects is `ssmparameter`.
```

Reply:
Thanks for noting the differences. I kept them as two separate procedures because I think that there are enough differences that it might get cluttered to note the differences.


I conditionalized each of these. Let me know how it looks now

Reply:
LGTM, but from preview page, it looks indeed much more redundant stuff :) . If it can be concise in some way while telling users the procedure clear, it would be better.

Reply:
Yes it definitely is redundant, but they can at least jump to whichever provider they're using and follow it start to finish without having to make too many decisions. 

I'll ask around a bit more to see what others think as well though